libr/anal/p/anal_x86_cs.c

```diff
@@ -2292,7 +2292,7 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len) {
 	{
 		ut64 naddr = addr;
 		size_t size = len;
-		if (insn == NULL)
+		if (!insn)
```

Comment:
well I wont comment any other curly braces issue comments on github are too slow so please fix all of them